### ISSUE TITLE
Material production/consumption for nrb not showing up in results

### ISSUE THREAD

**Issue by chauenstein** [YaleCIE/RECC-ODYM#64]
Hi @stefanpauliuk @CarrerF, maybe I am overseeing something, but could it be that materials used for renovation of non-residential buildings are not reported? 
(I wanted to look at F_6_7_ren for nrb, but it seems to me like no values are assigned for it in ODYM_RECC_Main (commit 7721b7d); I then also ran the model, once with renovation for nrb turned on, once turned off, and results reported for construction material production/consumption are the same in both runs [system wide GHG differ, also material footprint fossil fuels --> renovation has effect])
Thank you!

**Comment by CarrerF** [comment]
You are right, @chauenstein.
In the 30th April's commit renovation of non-res buildings has been introduced for energy savings potential, but not for material. 
Currently, 3_MC_RECC_Buildings_RECC in converted to 3_MC_RECC_Buildings_t to include renovation material, while 3_MC_RECC_NonResBuildings_RECC is not updated.

**Comment by stefanpauliuk** [comment]
Yes, we need some plausible data for the material requiremens of nrb renovation first :)
